### ISSUE TITLE
Terms

### ISSUE THREAD

**Issue by j-applese3d** [patcon/flashfood-api-docs#1]
Hi!

Thank you for this nice repository!
I'm just wondering on the legality of pulling flash-food data for customized alerts (ex, I want to be told ASAP when a certain product shows up at my local store)

Based on the FlashFood T&C, it looks like this might not be allowed? Do you know if using the API outside of the app is legal?

> By using our Services, you also agree **not to do** the following with respect to your account: ... **access or attempt to access your account using a proxy, client, or third party app**

From https://www.flashfood.com/en/terms, under "2   Accessing our services > Security"

**Comment by ChadMatsalla** [comment]
I was happily doing this and suddenly my host gets 403 Forbidden when I try to access api.flashfood.com. I saw no option but to contact flashfood and confess what I was doing in the hopes that they will change that. I can report back as to what they say.

**Comment by roblox28** [comment]
@ChadMatsalla Any updates on this?

**Comment by ChadMatsalla** [comment]
Still forbidden. Robotic auto reply from Flashfood support told me to “reinstall the app”.

**Comment by j-applese3d** [comment]
FWIW, I don't have any trouble (yet).
Which endpoints are you using?  Are you using the herokuapp URL or flashfood.com directly?

**Comment by patcon** [comment]
Yeah, I suspect that this is violating the terms and services. My hope was that if I was only doing very few queries, and only for personal use, no one would notice or care. I've done projects like this before, and once or twice I've gotten a DMCA take-down or an informal precursor to one (along with a friendly invitation to apply for a job), so I suppose there's a risk of this disappearing if it's usage is too flagrant 

I would recommend using a secondary account :)
